src/middleware.ts

```diff
@@ -0,0 +1,36 @@
+import type { RequestContext, ResponseContext, ConfigurationOptions, Middleware } from './gen/index.ts';
```

Comment:
I think this probably gets transpiled away since it is all types, but for consistency, can we do this:
```suggestion
import type { RequestContext, ResponseContext, ConfigurationOptions, Middleware } from './gen/index.js';
```